sql/codegen_alps.go

```diff
@@ -43,6 +43,7 @@ type alpsFiller struct {
 	Y      string
 
 	// Train
+	ImportCode        string
```

Comment:
maybe the name `CustomEstimatorImportCode` make more sence?

Reply:
I think ImportCode is better for it is more general. It's probable not limited to custom estimator imports In future.